install/ui/test/data/ipa_init.json

```diff
@@ -315,6 +315,18 @@
                             "view_certificate": "Certificate for ${entity} ${primary_key}",
                             "view_certificate_btn": "View Certificate"
                         },
+                        "certmap": {
```

Comment:
Yes, you're right. I will remove this change from current PR and will create new PR for removing offline WebUI.